pkg/network/ebpf/c/protocols/https.h

```diff
@@ -28,6 +30,25 @@ static __always_inline void https_process(conn_tuple_t *t, void *buffer, size_t
     read_into_buffer(http.request_fragment, buffer, len);
     http.owned_by_src_port = http.tup.sport;
     log_debug("https_process: htx=%llx sport=%d\n", &http, http.owned_by_src_port);
+
+    protocol_t *cur_fragment_protocol_ptr = bpf_map_lookup_elem(&dispatcher_connection_protocol, &http.tup);
+    if (cur_fragment_protocol_ptr == NULL) {
+        protocol_t cur_fragment_protocol = PROTOCOL_UNKNOWN;
+        conn_tuple_t inverse_conn_tup = http.tup;
+        flip_tuple(&inverse_conn_tup);
```

Comment:
can you explain why do you flip the conn tuple?

Reply:
If we haven't identified the protocol on our direction (haven't done that yet, or could not do at all), then we should try and see if we were able to classify the opposite direction

for example, "binary" protocols (gRPC, kafka, etc.) that can be identified in only one direction